wsmaster/che-core-api-workspace/src/main/java/org/eclipse/che/api/workspace/server/devfile/validator/DevfileSchemaValidator.java

```diff
@@ -75,6 +81,17 @@ private void validate(JsonNode contentNode) throws DevfileFormatException {
     try {
       List<Problem> validationErrors = new ArrayList<>();
       ProblemHandler handler = ProblemHandler.collectingTo(validationErrors);
+      if (!contentNode.hasNonNull("apiVersion")) {
+        throw new DevfileFormatException(
+            "Devfile schema validation failed. Error: The object must have a property whose name is \"apiVersion\".");
+      }
+      String apiVersion = contentNode.get("apiVersion").asText();
+
+      if (!schemasByVersion.containsKey(apiVersion)) {
+        throw new DevfileFormatException(
+            String.format("version '%s' of the devfile is not supported", apiVersion));
```

Comment:
Does it make sense to print supported versions here?

Reply:
it does. Updated